doc/installation/centos.rst

```diff
@@ -66,19 +66,23 @@ Create the virtual environment::
 
     $ virtualenv /opt/privacyidea
 
-and activate it::
+activate it::
 
     $ . /opt/privacyidea/bin/activate
 
+and install/update some prerequisites::
+
+    (privacyidea)$ pip install -U pip setuptools
+
 If this should be a pinned installation (that is the environment we use to build and test),
 we need to install some pinned dependencies first. They should match the version of the targeted
 privacyIDEA::
```

Comment:
We should add some always valid text like, "please check for the latest version of privacyIDEA" (where ot check!)
If it is "3.3" you need to issue the following commands. (Or use a tag $version).

Reply:
Done in 1354eaf